src/components/ShareThisPage/ShareThisPage.scss

```diff
@@ -21,12 +21,9 @@ $gw-colour-btn-share-this-page-border: $gw-colour-grey-2;
   border: 1px solid $gw-colour-btn-share-this-page-border;
   background-color: $gw-colour-btn-share-this-page-background;
 
-  // Class: 'gw-share-this-page__container'.
-  &__container {
-    display: inline-flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    width: auto;
+  details {
+    display: block;
+    display: -webkit-box; // Fixes the position in webkit browsers
```

Comment:
I'm not sure this work as it push the container 100% when open and not wrap the child content.

Reply:
Tested this in chrome and it's not allowing the child items to wrap.

Reply:
Interesting, I've cross browser tested it and it fixes the error for me. Do you have a screenshot you can share with me please?

Reply:
Here's the feature branch with it built, does it work for you here? https://GovWalesSecure!:28!wales%3DBOARD%3Dsignal99@govwales-search-gw10658-share-page.welshgovernment.wales/search